src/cozmo/annotate.py

```diff
@@ -307,7 +307,7 @@ def label_for_face(self, obj):
         '''
         expression = obj.known_expression
         if len(expression) > 0:
-            expression += " "
+            expression += "=%s%% " % obj.expression_score
```

Comment:
if there is a specific known expression, then also show the score (by appending it to the string, along with a % symbol to show that the value is out of 100 (`%%` is the escaped value for a `%` vs `%d` `%s` etc.)